packages/react/README.md

```diff
@@ -0,0 +1,101 @@
+# @reatom/react
+
+Package for bindings [ReAtom](github.com/artalar/reatom) store with React
+
+[![npm](https://img.shields.io/npm/v/@reatom/react?style=flat-square)](https://www.npmjs.com/package/@reatom/react)
+![npm type definitions](https://img.shields.io/npm/types/@reatom/react?style=flat-square)
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@reatom/react?style=flat-square)](https://bundlephobia.com/result?p=@reatom/react)
+![GitHub](https://img.shields.io/github/license/artalar/reatom?style=flat-square)
+
+
+## Install
+
+Yarn
+```
+yarn add @reatom/react
+```
+
+NPM
+```
+npm i -S @reatom/react
+```
+
+## Hooks Api
+
+
+### useAtom
+Connects the atom to the store represented in context and returns the state of the atom from the store (or default atom state)
+
+```ts
+useAtom(atom: Atom, silent: boolean = true): AtomState
```

Comment:
I don't know how best to label the type of return. That's why I called him that `AtomState`....